applications/RomApplication/custom_python/rom_python_application.cpp

```diff
@@ -47,6 +47,7 @@ PYBIND11_MODULE(KratosRomApplication,m)
 
     //registering variables in python
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, ROM_BASIS )
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, ROM_LEFT_BASIS )
```

Comment:
Would it make sense to also rename the original "ROM_BASIS" as "ROM_RIGHT_BASIS" ?

Reply:
I will write it in a TODO list for future PR.